app/uk/gov/hmrc/incometaxpenaltiesfrontend/controllers/IndexController.scala

```diff
@@ -64,7 +64,8 @@ class IndexController @Inject()(override val controllerComponents: MessagesContr
           lspCardData = lspSummaryCards,
           lppCardData = lppSummaryCards,
           penaltiesOverviewViewModel = PenaltiesOverviewViewModel(penaltyData),
-          isAgent = penaltyDataUserRequest.isAgent
+          isAgent = penaltyDataUserRequest.isAgent,
+          actionsToRemoveLinkDate = penaltyData.lspPeriodOfComplianceDate
```

Comment:
```suggestion
          actionsToRemoveLinkDate = penaltyData.lspPeriodOfComplianceDate.getOrElse(LocalDate.now)
```

Would be nice to keep the logic within the controller rather than the view, can then change the types to `LocalDate` instead of `Option[LocalDate]`